src/Features/Core/Portable/Completion/CompletionService.ProviderManager.cs

```diff
@@ -98,6 +96,9 @@ ImmutableArray<CompletionProvider> GetImportedAndBuiltInProvidersWorker(Immutabl
                 }
             }
 
+            /// <summary>
+            /// Don't call. For test only.
+            /// </summary>
```

Comment:
💡 This can move to a test accessor with two benefits:

1. The test method is constrained to a test method location
2. We have an analyzer to enforce that test accessors are only used by test code

Reply:
fixed